CHANGELOG.md

```diff
@@ -112,6 +112,7 @@ All notable changes to this project will be documented in this file. The format
     - **Note**: the previous `Tab` child of `TabNav` has been renamed to `TabNavItem`
   - Icon support for standard variants of Button component ([#1268](https://github.com/bloom-housing/bloom/pull/1268)) (Jared White)
   - Generic date component ([#1392](https://github.com/bloom-housing/bloom/pull/1392)) (Emily Jablonski)
+  - Add ActionBlock component ([#1404](https://github.com/bloom-housing/bloom/pull/1459)) (Marcin Jedras)
```

Comment:
I think this may be in the wrong place in the changelog? I just put mine [at the top](https://github.com/bloom-housing/bloom/pull/1459/files#diff-06572a96a58dc510037d5efa622f9bec8519bc1beab13c9f251e97e657a9d4edR18) under unreleased

Reply:
You are right, cheers.